.travis.yml

```diff
@@ -10,11 +10,8 @@ node_js:
 
 before_script:
```

Comment:
It's not a mistake. Master now prints the version just in case. Put those back in please :smiley:

Reply:
Yeah, I was trying everything to fix the build; turned out it was a cache issue.